### ISSUE TITLE
Missing 'X-Per-Page' header from server

### ISSUE THREAD

**Issue by gumutianqi** [gmessner/gitlab4j-api#286]

**Comment by gumutianqi** [comment]
> use gitlab api V3

```
Caught: org.gitlab4j.api.GitLabApiException: Missing 'X-Per-Page' header from server
org.gitlab4j.api.GitLabApiException: Missing 'X-Per-Page' header from server
	at org.gitlab4j.api.Pager.getHeaderValue(Pager.java:114)
	at org.gitlab4j.api.Pager.<init>(Pager.java:96)
	at org.gitlab4j.api.TagsApi.getTags(TagsApi.java:70)
	at org.gitlab4j.api.TagsApi.getTags(TagsApi.java:39)
	at org.gitlab4j.api.TagsApi$getTags.call(Unknown Source)
	at me.weteam.dayone.jenkins.gitlab-api.run(gitlab-api.groovy:31)
```

**Comment by gmessner** [comment]
@gumutianqi 
This is actually a bug with the GitLab server v3 API, I have however added handling of this condition to this library and released 4.9.11

Please verify and close this issue when you have time.

**Comment by gumutianqi** [comment]
thanks, It works, the released 4.9.11 fixed my problem.
